cfgov/jinja2/v1/_includes/organisms/featured-content.html

```diff
@@ -6,94 +6,56 @@
 
    Description:
 
-   See [GHE]/CFPB/hubcap/wiki/Featured-Content-Module
+   See https://cfpb.github.io/design-system/components/featured-content-module
 
    Create a featured content molecule when given:
 
-   value:                 Object defined from a StreamField block.
+   value.heading:   Heading text.
 
-   value.heading:         Heading text.
+   value.body:      Body text (rich text).
```

Comment:
I promise I won't forget. We'll need to edit this file anyway to remove the `| richtext`.

Reply:
Right, I'm just saying it would be easy to remove the `| richtext` and forget to remove this parenthetical in the opening comment. But not a huge deal.